apps/client/src/pages/ResultsPage.js

```diff
@@ -75,55 +78,60 @@ const ResultsPage = ({ topic, checker }) => {
           <UserAnswer strong>Uw antwoord</UserAnswer>
           <Change />
         </MainWrapper>
-
         <div>
           {checker?.stack?.map((question, index) => {
             const isDecisiveForPermits =
               uniqBy(permitsPerQuestion[index], "name") || [];
-            return (
-              <QuestionWrapper key={question.id}>
-                <Question>{question.text}</Question>
-                {question.options ? (
-                  <UserAnswer>
-                    {question.answer.replace(/['"]+/g, "")}
-                  </UserAnswer>
-                ) : (
-                  <UserAnswer>
-                    {
-                      booleanOptions.find(
-                        (option) => option.value === question.answer
-                      ).label
-                    }
-                  </UserAnswer>
-                )}
-                <Change>
-                  <Button
-                    onClick={() => onGoToQuestion(index)}
-                    variant="textButton"
-                  >
-                    Wijzig
-                  </Button>
-                </Change>
 
-                {isDecisiveForPermits.map((permit, index) => (
-                  <UserResult key={`${permit} ${index}`}>
-                    <Icon
-                      color="secondary"
-                      size={30}
-                      style={{
-                        flexShrink: 0, // IE11 Fix
-                      }}
+            if (
+              typeof question.answer === "boolean" ||
+              typeof question.answer === "string"
+            ) {
```

Comment:
Why add this check?

Reply:
To make sure the questions are always set.